try:
    # Code that might raise an exception
    # ...
except ExceptionType:
    # Code to handle the exception
    # ...


    try:
    num = int(input("Enter a number: "))
    result = 10 / num
    print("Result:", result)
except ZeroDivisionError:
    print("Error: Division by zero!")
except ValueError:
    print("Error: Please enter a valid number!")

   